docs/tasks/administer-federation/cluster.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Federated Cluster
+redirect_from:
+- "/docs/user-guide/federation/cluster/"
+- "/docs/user-guide/federation/cluster.html"
+---
+
+{% capture overview %}
+
+This guide explains how to use Clusters in a Federation control plane.
+
+Different than other Kuberentes resources, such as Deployments, Services and ConfigMaps,
+clusters only exist in the federation context, i.e. those requests must be submitted to the
+federation api-server.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include federated-task-tutorial-prereqs.md %}
+* You should also have a basic [working knowledge of Kubernetes](/docs/setup/pick-right-solution/) in
+general.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Listing Clusters
+
+To list the clusters available in your federation, you can use [kubectl](/docs/user-guide/kubectl/) by
+running:
+
+``` shell
+kubectl --context=federation-cluster get clusters
+```
+
+The `--context=federation-cluster` flag tells kubectl to submit the
+request to the Federation apiserver instead of sending it to a Kubernetes
+cluster. If you submit it to a k8s cluster, you will receive an error saying
+`the server doesn't have a resource type "clusters"`
+
+If you passed the correct Federation context but received the error message above, it means that you haven't
+added any cluster to the Federation yet.
```

Comment:
Is this true? If you are hitting the federation endpoint and if you don't have any clusters, you should get `No resources found.`